builder/azure/arm/config.go

```diff
@@ -562,6 +562,8 @@ type Config struct {
 	LicenseType string `mapstructure:"license_type" required:"false"`
 	// Specifies if Secure Boot and Trusted Launch is enabled for the Virtual Machine.
 	SecureBootEnabled bool `mapstructure:"secure_boot_enabled" required:"false"`
+	// Specifies if Encryption at host is enabled for the Virtual Machine.
```

Comment:
```suggestion
	// Specifies if Encryption at host is enabled for the Virtual Machine.
	// Requires enabling encryption at host in Azure read more [here](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-host-based-encryption-portal?tabs=azure-powershell)
```

Reply:
@kamilpi do you mind taking in this change as well? Afterwards you need to run `make generate` to generate the docs site from the config, let me know if you have any trouble with this as I can just commit the change to your fork

Reply:
Done.